go.mod

```diff
@@ -13,8 +13,9 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.26.0
+	github.com/shirou/gopsutil v3.21.8+incompatible // indirect
```

Comment:
Did this come along for the ride or was it intentional?

Reply:
:thinking: let me go mod tidy again

Reply:
yeah, `go mod tidy` added this, probably as a result of me using a newer Go version